cpp/order/wigner3j.h

```diff
@@ -16,32 +16,40 @@ vector<float> getWigner3j(unsigned int l);
 
 // All wigner3j coefficients created using sympy
 /*
+
 from sympy.physics.wigner import wigner_3j
-import scipy.io as sio
 import numpy as np
 import json
 
 wigner = {}
-for i in range(10):
-    counter = 0
-    l = (i+1)*2
-    print('Computing l={}'.format(l))
-    for u1 in range(2*l+1):
-        for u2 in range(max(l-u1,0),min(2*l+1,3*l-u1+1)):
-            counter += 1
-    W_l = np.zeros(counter, dtype=np.float64)
-
-    j = 0
-    for u1 in range(2*l+1):
-        for u2 in range(max(l-u1,0),min(2*l+1,3*l-u1+1)):
-            u3 = 3*l-u2-u1
-            W_l[j] = float(wigner_3j(l,l,l,u1-l,u2-l,u3-l))
-            j += 1
-    wigner['l'+str(l)] = W_l.tolist()
+for l in range(21):
```

Comment:
This new (commented) Python script can be used to regenerate the Wigner 3j coefficients with greater automation than the previous version. Almost all the code in `wigner3j.cc` is generated by this script. I regenerated that file because I didn't trust its results at first and also wanted to add the ability to compute Wl for small (0, 1) and odd (3, 5, 7, ..., 19) values of l.